src/finder/frameworkSpecs.ts

```diff
@@ -0,0 +1,109 @@
+import { FrameworkSpec } from "./types";
+
+export const frameworkSpecs: FrameworkSpec[] = [
+  {
+    id: "core:express",
+    runtime: "nodejs",
+    webFrameworkId: "Express.js",
+    requiredDependencies: [
+      {
+        name: "express",
+      },
+    ],
+  },
+  {
+    id: "angular",
+    runtime: "nodejs",
+    webFrameworkId: "Angular",
+    requiredFiles: ["next.config.js", "next.config.ts"],
+    requiredDependencies: [
+      {
+        name: "@angular/core",
+      },
+      {
+        name: "@angular/cli",
+      },
+    ],
+    commands: {
+      build: {
+        cmd: "ng build",
```

Comment:
npm run prerender || npm run build:ssr || npm run build

Reply:
Is it just one of these is sufficient here?